docs/Getting-Started-with-Balance-Ball.md

```diff
@@ -267,7 +267,7 @@ for each training run. In other words, "BalanceBall1" for the first run,
 every training run are saved to the same directory and will all be included 
 on the same graph.
 
-To summarize, go to your command line, enter the `ml-agents` directory and type: 
+To summarize, go to your Anaconda command line, activate the ml-agents environment, enter the `ml-agents` directory and type: 
```

Comment:
Ok I've made that change. It's just that the previous docs made us set up with Anaconda, so I was under the impression that it would assume we'd use it throughout.